base/io.jl

```diff
@@ -448,7 +456,32 @@ function readuntil(s::IO, t::AbstractString)
 end
 
 readline() = readline(STDIN)
-readline(s::IO) = readuntil(s, '\n')
+
+function readline(s::IO, chomp = false; nl2lf=false)
```

Comment:
There is `readuntil(s::IO, t::AbstractString)`, but I would expect this to be more efficient. It's modified from readuntil.